cmd/logger/logger.go

```diff
@@ -314,7 +314,19 @@ func LogIf(ctx context.Context, err error) {
 	fmt.Println(output)
 }
 
+// CriticalIf :
+// Like LogIf with exit
+// It'll be called for fatal error conditions during run-time
+func CriticalIf(ctx context.Context, err error) {
```

Comment:
Can the `LogIf` be moved into the `if` section that way we do not check for err not being nil twice.
```
func CriticalIf(ctx context.Context, err error) {
	
	if err != nil {
               LogIf(ctx, err)
		os.Exit(1)
	}
}```

Reply:
Done.